### PR TITLE
serialization: addition of files marcxml field

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by krzysztof** [comment]
@lnielsen the leader field is missing in this PR, it also does not appear in the marcxml export in legacy, was this ever exported?

**Comment by krzysztof** [comment]
Closes #914

File: zenodo/modules/records/serializers/to_marc21/rules.py [hunk 1]
```diff
@@ -107,3 +107,18 @@ def reverse_conkerence_url(dummy_self, key, value):
...
+
+@to_marc21.over('856', '^_files$')
+@utils.reverse_for_each_value
+@utils.filter_values
+def reverse_files(dummy_self, key, value):
+    """Reverse - Files."""
+    return {
+        's': str(value['size']),
+        'u': value['uri'],
+        'w': value['checksum'],
```
Comment (lnielsen) [inline zenodo/modules/records/serializers/to_marc21/rules.py]: In the old MARC export we didn't include the checksum and using [**$w - Record control number**](https://www.loc.gov/marc/bibliographic/bd856.html) I don't think would be correct.

Also, we did add **$z - Public note** with the ordering of the item, but I think we can leave it out.

Comment (krzysztof) [inline zenodo/modules/records/serializers/to_marc21/rules.py]: Yes, I misinterpreted it - I'll remove the $w.
Should the checksum be in $z perhaps? I think checksum is more important to include than ordering?

**Comment by krzysztof** [comment]
marcxml export: Adds leader field and files for open-access records.

**Comment by krzysztof** [comment]
also needs communicating that $z in files will now return the md5 checksum

**Comment by krzysztof** [comment]
@lnielsen Can you take another look?

**Pull request merged**